File: Csharp/Find Multiples of a Number.cs
// https://www.codewars.com/kata/58ca658cc0d6401f2700045f

using System.Linq;
using System.Collections.Generic;

public class Kata
{
  public static List<int> FindMultiples(int num, int limit) =>
     Enumerable.Range(1, limit / num)
               .Select(x => x * num)
               .ToList();
 }

File: javascript/Find Multiples of a Number.js
// https://www.codewars.com/kata/58ca658cc0d6401f2700045f/train/javascript

function findMultiples(integer, limit) {
    let result = []
    for(i = integer; i<=limit; i+=integer){
      result.push(i)
    }
      return result
    }